docs/datastores.md

```diff
@@ -41,6 +28,19 @@ db:
 
 Ensure that `dbdir` is not included in the config.
 
+NeDB End-of-life
```

Comment:
Why shift this down ooi? Shouldn't it be prominent for people who want to upgrade.

Reply:
Everyone, new or upgrading will need the postgres section, and we link to it during the setup guide, so I thought that should be the most prominent. It's probably worth having a specific page on upgrading from pre-1.0